validation/src/main/scala/uk/gov/nationalarchives/tdr/validation/utils/ConfigUtils.scala

```diff
@@ -0,0 +1,166 @@
+package uk.gov.nationalarchives.tdr.validation.utils
```

Comment:
The `"alternateKeys"` and `"properties"` strings are used in multiple places. Maybe add them as object level vals?

Reply:
added at object level